docs/operation-guide.md

```diff
@@ -11,21 +11,68 @@ $ namespace="tidb"
 
 > **Note:** The rest of the document will use `values.yaml` to reference `charts/tidb-cluster/values.yaml`
 
+## Configuration
+
+TiDB Operator use `values.yaml` as TiDB cluster configuration file. It provides the default basic configuration which you can use directly for quick deployment, but if you have specific configuration requirements or for production deployment, you need to manually modify the variables in the `value.yaml` file.
+
+* Resource setting
+
+    * CPU & Memory
+
+        The default deployment doesn't set CPU and memory requests or limits for any of the pods, these settings can make TiDB cluster run on a small Kubernetes cluster like DinD or the default GKE cluster for testing. But for production deployment, you would likely to adjust the cpu, memory and storage resources according to the [recommendations](https://github.com/pingcap/docs/blob/master/op-guide/recommendation.md).
```

Comment:
https://github.com/pingcap/docs/blob/master/op-guide/recommendation.md

this link is broken.